README.md

```diff
@@ -22,16 +42,26 @@ ts_m2fpga
 │   ├── ilcCommmunication.md
 ├── fpgaInterface
 │   ├── NiFpga_mainFPGA.h
-│   ├── NiFpga_mainFPGA.lvbitx
 │   ├── NiFpga.c
 │   ├── NiFpga.h
+│   ├── NiFpga_mainFPGA.lvbitx
+│   ├── bitfile
+│       ├── m2fpga_FPGATarget_mainFPGA_<random_characters>.lvbitx
 ├── README.md
 └── .gitignore
 ```
 
 - `src`: contains the LabVIEW FPGA project.
 - `doc`: documentation like `versionHistory.md`.
 - `fpgaInterface`: in here are all files generated by the FPGA Interface C API.
+- `bitfile`: contains the bitfile (.lvbitx).
```

Comment:
`fpgaInterface/bitfile`: contains the bitfile (`.lvbitx`).

Reply:
Done.